Pract. 02/Solutions/Task12.cpp

```diff
@@ -0,0 +1,42 @@
+﻿// Task12.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int day, month;
+    cout << "Enter a day: ";
+    cin >> day;
+    cout << "Enter a month: ";
+    cin >> month;
+    if (!(day == 30 || day == 31 || (day == 28 && month == 2))) {
+        cout << "Next date: " << ++day << " " << month;
+    }
+    else if ((day == 30 && month % 2 == 0) || (day == 31 && month % 2 != 0)) {
```

Comment:
Don't forget that November can't be divided by 2 and has 30 days. Also, don't forget about 31st of December.